monai/networks/nets/swin_unetr.py

```diff
@@ -1055,3 +1055,34 @@ def forward(self, x, normalize=True):
         x4 = self.layers4[0](x3.contiguous())
         x4_out = self.proj_out(x4, normalize)
         return [x0_out, x1_out, x2_out, x3_out, x4_out]
+
+
+def filter_swinunetr(key, value):
```

Comment:
perhaps in the docstring we can mention that this function is typically used with `copy_model_state`

Reply:
Thanks Wenqi, have mentioned it in the docstring.